plugins/inventory/nb_inventory.py

```diff
@@ -943,6 +944,12 @@ def extract_cluster(self, host):
         except Exception:
             return
 
+    def extract_cluster_device(self, host):
+        # cluster device does not have a slug
+        if host.get("device", None) is None:
+            return ""
```

Comment:
Are you certain this is the correct thing to return? This doesn't seem like a good idea to me